### PR TITLE
[OGM] Update FAQ about missing Hibernate Search on the class path

### PR DESCRIPTION
**Pull request by DavideD**
I've been asked this question a couple of time already.
I think we also need to explain better when HIbernate Search is optional or not in the documentation.

@Sanne or @gsmet WDYT?

### CODE REVIEW

File: ogm/faq.adoc [hunk 1]
```diff
@@ -183,6 +183,27 @@ Dependencies: org.hibernate:ogm services, org.hibernate.ogm.mongodb services
 Alternatively, you can configure this via the descriptor _jboss-deployment-structure.xml_.
 See the https://docs.jboss.org/hibernate/ogm/4.1/reference/en-US/html_single/#ogm-configuration-jbossmodule[reference guide] to learn more.
 
+=== Why do I get a `java.lang.NoClassDefFoundError: org/hibernate/search/batchindexing/spi/MassIndexerFactory`?
+
+This means that you need Hibernate Search on the class path.
```
Comment (gsmet) [inline ogm/faq.adoc:188]: s/class path/classpath/

File: ogm/faq.adoc [hunk 2]
```diff
@@ -183,6 +183,27 @@ Dependencies: org.hibernate:ogm services, org.hibernate.ogm.mongodb services
...
+[source,xml]
+----
+<dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-search-orm</artifactId>
+</dependency>
+----
+
+You can include the right version using the BOM as describe in the
+https://docs.jboss.org/hibernate/stable/ogm/reference/en-US/html/ch02.html[documentation getting started].
```
Comment (gsmet) [inline ogm/faq.adoc:200]: "documentation getting started" looks weird. "Getting started chapter of the documentation" maybe?

**Comment by gsmet** [comment]
Yes, I've also seen this question so it looks like a good addition.

And I agree with you, we probably need to be more clear in the documentation.

**Comment by DavideD** [comment]
Applied the remarks and merged https://github.com/hibernate/hibernate.org/pull/110

**Pull request closed without merging**